### ISSUE TITLE
Getting an extraneous CR on CREATE TABLE line on OSX if original dump is CRLF.

### ISSUE THREAD

**Issue by changeling** [dumblob/mysql2sqlite#72]
```
awk version 20070501
Mac OS X 10.15.1
```

I'm seeing CRLF on CREATE TABLE opening lines, and only those lines, with windows-created (CRLF) files converted on OSX. The CRLF is replaced with (correctly per awk's RS) LF on subsequent lines within each CREATE statement, but reappears on each subsequent CREATE.

Likely not a big deal, but for the sake of detail, I thought I'd point it out.

Example:
```
CREATE TABLE IF NOT EXISTS `myveryowntable` (<CRLF>
  `ID` INT(11) NOT NULL AUTO_INCREMENT,<CRLF>
  `User_ID` INT(11) NOT NULL,<CRLF>
  PRIMARY KEY (`ID`)<CRLF>
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;<CRLF>
```

```
CREATE TABLE IF NOT EXISTS `myveryowntable` (<CRLF>
  `ID` integer NOT NULL PRIMARY KEY AUTOINCREMENT<LF>
,  `User_ID` integer NOT NULL<LF>
);<LF>
```

PS Also note that commas are moved to follow the new line. This is independent of the newline character in the original dump. Not a real problem, just an aesthetic note.

**Comment by dumblob** [comment]
Yeah, it's known, that CRLF is not treated in a special way - sqlite is very generous in this regard, so I don't think it's something we should change :wink:.

>PS Also note that commas are moved to follow the new line. This is independent of the newline character in the original dump. Not a real problem, just an aesthetic note.

Yes, this is on purpose (it simplifies the script considerably). The output shall be a valid sqlite input and not something "pleasant to see" :wink:.

Feel free to close this issue if you don't have any actual practical issues with that.
